scalafmt-core/shared/src/main/scala/org/scalafmt/config/DanglingParentheses.scala

```diff
@@ -3,32 +3,37 @@ package org.scalafmt.config
 import metaconfig._
 
 case class DanglingParentheses(
-    callSite: Boolean,
-    defnSite: Boolean
+    callSite: Option[Boolean] = None,
```

Comment:
I see only `!style.danglingParentheses.defnSite.isFalse` and `!cfg.defnSite.isFalse` usage which means "true or None" i.e. default value in this PR. Did I miss something?

And in follow up of https://github.com/scalameta/scalafmt/pull/1696 discussion: when the user sees the default value `danglingParentheses = true` in the documentation, he will not explicitly set it in the settings. Maybe it makes sense to not introduce third state of config (`None`) and just its implement behavior for `true` state?

Reply:
Done. The `Option` is removed.